pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -46,14 +55,26 @@ pub mod pallet {
 	/// If true, it means that LocalAssets storage has been removed.
 	pub(crate) type LocalAssetsMigrationCompleted<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+	#[pallet::storage]
+	/// The total number of suicided contracts that were removed
+	pub(crate) type SuicidedContractsRemoved<T: Config> = StorageValue<_, u32, ValueQuery>;
```

Comment:
What's the value of storing this count on state?

Reply:
The purpose of this storage is to keep track of how many contracts have already been migrated.